manifests/r/RemixDev/deemix-gui/2022.5.28-r206.a752a63e0e.20220528.1/RemixDev.deemix-gui.locale.en-US.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: RemixDev.deemix-gui
+PackageVersion: 2022.5.28-r206.a752a63e0e.20220528.1

```

```suggestion
PackageVersion: 2022.5.28-r206.a752a63e0e
```
To match Control Panel which will prevent upgrade loops.